USING_VANAGON.md

```diff
@@ -0,0 +1,42 @@
+# Using Vanagon
+- Note that these docs are incomplete. Docs for all parts of vanagon are on their way
+
+## Creating a Vanagon Project for Windows
+
+### Overview
+Vanagon projects for windows use [WiX](http://wixtoolset.org/) to create MSIs that will install on native windows platforms.  Vanagon uses the same process between all platforms, including windows, to build and stage artifacts before creating an installer.  Vanagon only uses WiX files for the actual installer itself, so the smaller / simpler your install experience, the less WiX you need. You can find example WiX files under [examples](https://github.com/puppetlabs/vanagon/examples/resources/windows/wix).  These files should provide a good start to create an MSI using vanagon.
```

Comment:
I wasn't sure about putting these under examples - would prefer that actually put in generic files with plenty of comments and the instruction that if you wanted to modify them, copy them to the project specific are and then modify.

You need to make clear that the files under the Vanagon resources are should not be modified, but are over-ridden by anything in the project specific resources area.